addons/web/static/src/webclient/commands/command_palette.js

```diff
@@ -18,7 +18,19 @@ const commandCategoryRegistry = registry.category("command_categories");
 const commandProviderRegistry = registry.category("command_provider");
 
 /**
- * @typedef {import("./command_service").Command} Command
+ * @typedef {{
+ *  action: ()=>void || NextProvider,
```

Comment:
This should be written like that in order to work properly
```suggestion
 *  action: ()=>(void | NextProvider),
```